tests/unit/computeds/calendar-test.js

```diff
@@ -1,46 +1,53 @@
-import $ from 'jquery';
 import EmberObject from '@ember/object';
-import { getOwner } from '@ember/application';
 import moment from 'moment';
-import { moduleFor, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
 import calendar from 'ember-moment/computeds/calendar';
 import tz from 'ember-moment/computeds/tz';
 import locale from 'ember-moment/computeds/locale';
 
-moduleFor('ember-moment@computed:moment', {
-  setup() {
-    this.register('object:empty', EmberObject.extend({}));
-    moment.locale('en');
-  }
-});
+module('ember-moment@computed:moment', function (hooks) {
+  setupTest(hooks);
+
+  hooks.beforeEach(function () {
+    this.setup = () => {
+      this.owner.register('object:empty', EmberObject.extend({}));
+      moment.locale('en');
+    };
+    this.createSubject = (attrs) => {
```

Comment:
I don't know if it could help, or maybe this is just not compatible for the targeted versions, but in my pr, I rewrote this like this, and put it in an helper (as it was copy/pasted in several tests)
```
export function createSubject(attrs) {
  const factory = this.owner.factoryFor('object:empty').class.extend(attrs);
  return factory.create();
}
```

Reply:
Can you link to your PR? I may want to take what you already done and apply it here :D